cli/index.ts

```diff
@@ -519,9 +530,9 @@ const builtYargs = createYargsCli({
         };
 
         runner.onChange(printExecutedGraph);
-        const runResult = await runner.resultPromise()
+        const runResult = await runner.resultPromise();
         printExecutedGraph(runResult);
-        return ((runResult.status === dataform.RunResult.ExecutionStatus.FAILED) ? 1 : 0);
+        return runResult.status === dataform.RunResult.ExecutionStatus.FAILED ? 1 : 0;
```

Comment:
suggest inverting this to only return `0` when the status is `SUCCESSFUL`; that way all other statuses are guaranteed to return `1` (which is probably more future-compatible with new statuses, or at least more safely compatible with them)

Reply:
Ah, good spot :)